### DISCUSSION TITLE
In admin dashboard, how to make it "search by name" instead of "search by id"?

### DISCUSSION THREAD

**Discussion by deltavin** [payloadcms/payload#500] [Q&A]
In the dashboard, there is an option to search by id. How can I change it to search by another field, say "name"? I see this happening in the live payload demo at https://demo.payloadcms.com/ (screenshot attached).

Is it a configuration setting or is it a custom component for the dashboard?

Also, where can I find the source code for the demo?

![pcms](https://user-images.githubusercontent.com/36921075/161440234-fdfa43bc-eb4d-461c-9115-c6ab7e75892f.png)

Thanks

**Comment by denolfe** [comment]
Hey @deltavin, great question. The demo is using the `useAsTitle` under collection admin options. Documentation on that can be found [here](https://payloadcms.com/docs/configuration/collections#admin-options). The demo uses the `defaultColumns` configuration as well. Here is the `admin` section of the Posts collection in the demo:

```ts
  admin: {
    defaultColumns: ['title', 'author', 'category', 'tags', 'status'],
    useAsTitle: 'title',
  },
```

I hope that helps. Let me know if you need any additional clarification. We intend to make the demo's source code public in the near future 👍

**Reply by deltavin** [reply]
Hey denolfe, 
Thanks for responding. 

That's exactly what I have in my config, but it didn't work.

This is my "people" collection: 
```
import { CollectionConfig } from "payload/dist/collections/config/types";
import Basics from "../customFields/Basics";

const People: CollectionConfig = {
  slug: "people",
  labels: {
    singular: "Person",
    plural: "People",
  },
  admin: {
    defaultColumns: ["name", "slug", "workTitle"],
    useAsTitle: "name",
  },
  fields: [
    {
      type: "row",
      label: "Basics",
      fields: Basics,
    },
    {
      name: "workTitle",
      label: "Work title",
      type: "text",
    }
  ],
};

export default People;

```

And, this is what it looks like in the admin dashboard. 

![people](https://user-images.githubusercontent.com/36921075/161441248-981a9b72-c0f3-4824-ac4b-f3a67344b823.png)


Could it be because the fields "name" and "slug" are a part of a custom field group called "Basics"?

This is my "Basics" const: 
```
import { Field } from "payload/dist/collections/config/types";
import { Slug } from "./Slug";

const Basics: Field[] = [
  {
    name: "name",
    label: "Name",
    type: "text",
    required: true,
    admin: {
      width: "50%",
    },
  },
  {
    name: "slug",
    label: "Slug",
    type: "text",
    required: true,
    index: true,
    unique: true,
    admin: {
      width: "50%",
      components: {
        Field: Slug,
      },
    },
  },
  {
    name: "intro",
    label: "Intro",
    type: "text",
    admin: {
      width: "100%",
    },
  }
];

export default Basics;

```

**Reply by jmikrut** [reply]
Hey @deltavin — yes, this is exactly why this is happening. It's because your Basics fields are within a `row`. We should be able to update this so that you can use fields within `row`s as your `useAsTitle`.

Give us a bit here and we'll see what we can do!
